clang/test/3C/hasgeneric.c

```diff
@@ -0,0 +1,8 @@
+// RUN: true
```

Comment:
Can you just add this to the existing `canwrite_constraints.h` file instead?  Also, to simplify any future troubleshooting and make clear that the problem isn't specific to `malloc`, please declare your own generic function rather than pulling in all of `stdlib.h`.

Reply:
Sure

Reply:
It doesn't look like this happened in this PR.  Can you please go back and do it now?  (Maybe I should have used the "Request changes" status...)

Reply:
This is being addressed in #670 now.